src/org/loklak/harvester/BaseScraper.java

```diff
@@ -107,11 +114,16 @@ public Post getDataFromConnection(String url, String type) throws IOException {
         return postArray;
 
     }
-/*
+
     public Post getDataFromConnection(String url) throws IOException {
         return getDataFromConnection(url, "all");
     }
-*/
+
+    public Post getDataFromConnection() throws IOException {
+        String url = this.prepareSearchUrl("all");
```

Comment:
there is a need of argument to be passed in prepareSearchUrl method, if there is no `type` argument, there is need to have an argument, `all` appears to fit here.